modules/migration-installing-cam-operator-ocp-3.adoc

```diff
@@ -54,14 +54,16 @@ $ sudo podman login registry.redhat.io
 +
 [source,terminal,subs="attributes+"]
 ----
-$ sudo podman cp $(sudo podman create registry.redhat.io/rhmtc/openshift-migration-rhel7-operator:v{mtc-version-z}):/operator.yml ./
+$ sudo podman cp $(sudo podman create \
+  registry.redhat.io/rhmtc/openshift-migration-rhel7-operator:v{mtc-version}):/operator.yml ./
```

Comment:
@apinnick I am curious why you split this and  the following command into two lines. Is there a benefit to presenting the command this way?

Reply:
I broke it up because, when it was a single line, it was too long for the user to see without scrolling sideways.